components/NavigationBar/src/ColorThemer/MDCNavigationBarColorThemer.h

```diff
@@ -17,20 +17,54 @@
 #import "MaterialColorScheme.h"
 #import "MaterialNavigationBar.h"
 
+/**
+ The possible theming variants that can be applied to a navigation bar.
+ */
+typedef NS_ENUM(NSInteger, MDCNavigationBarColorThemerVariant) {
```

Comment:
Why do we need an enum instead of having 2 different methods for the (currently) 2 different variants?

```
- applyPrimaryVariantColorScheme
- applySurfaceVariantColorScheme
```

Reply:
Dropped the enum. Had been flip-flopping on which approach to pursue.